x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -9709,17 +9709,9 @@
     "xpack.lens.xySuggestions.stackedChartTitle": "スタック",
     "xpack.lens.xySuggestions.unstackedChartTitle": "スタックが解除されました",
     "xpack.lens.xySuggestions.yAxixConjunctionSign": " と ",
-    "xpack.lens.xyVisualization.areaLabel": "エリア",
```

Comment:
Why this removals here? I see these labels are still used.

Reply:
You are right, I changed the approach and now area, bar, line, stacked area and stacked bar don't change. I will add them back